### PR TITLE
Add AWS topic

### PR DESCRIPTION
**Pull request by dmfrancisco**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

It's in the `topics-todo.md` file, as one of the popular topics that lack context 🙂

### CODE REVIEW

File: topics/aws/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+display_name: AWS
```
Comment (nayafia) [inline topics/aws/index.md]: hmm...we should probably spell this out as `Amazon Web Services`

**Comment by nayafia** [comment]
@dmfrancisco hooray! Thank you! Left one comment in review, after that we should be good to go 👍

**Comment by dmfrancisco** [comment]
@nayafia Thanks, just made the change 👍

**Comment by nayafia** [comment]
@dmfrancisco merged! 🎉

**Pull request merged**